Makefile

```diff
@@ -163,6 +163,19 @@ kfserving-install: knative-install cert-manager-install
 seldon-install: istio-install
 	@./scripts/seldon-operator-install.sh
 
+new-k8s-cluster:
```

Comment:
I would probably name it like 'new-test-cluster' so the purpose is clearer.

Reply:
done.